Issue/#6 Create a continuous deployment pipeline

wooneh opened the pull request:
Fixes #6 

## Proposed Changes

  - Automatically deploy to the cloud once a pull request is merged
  - Deploys frontend and backend separately
  - App is located at https://se701group6-assignment1.herokuapp.com/

The API key should be added to Secrets under the repo settings (I don't have access to this) so that GitHub Actions can use it.

timovv commented:
Cool, I've added you as project admin so you should be able to add the secret. Probably better that way then you sending me the API key :)

wooneh commented:
> Cool, I've added you as project admin so you should be able to add the secret. Probably better that way then you sending me the API key :)

Thanks, I've added it now so it should be good to go
